processing/src/main/java/org/apache/druid/segment/nested/ScalarDoubleColumnAndIndexSupplier.java

```diff
@@ -128,6 +135,12 @@ public static ScalarDoubleColumnAndIndexSupplier read(
           );
         }
 
+        final CompressedVSizeColumnarIntsSupplier encodedCol = CompressedVSizeColumnarIntsSupplier.fromByteBuffer(
+            encodedValuesBuffer,
+            byteOrder,
+            columnBuilder.getFileMapper()
+        );
```

Comment:
seems unfortunate we have to read this since we aren't using it anywhere, which is why it was left out previously